examples/run_nutation.py

```diff
@@ -61,48 +61,30 @@ def API_sender(value):
     API_sender(B0)
 #}}}
 date = datetime.now().strftime('%y%m%d')
-output_name = 'TEMPOL_capProbe'
-adcOffset = 25
-carrierFreq_MHz = 14.896101
+output_name = 'TEMPOL_capillary_probe_nutation_1'
+ph1_cyc = r_[0,2]
+ph2_cyc = r_[0,2]
+nPhaseSteps = len(ph1_cyc)*len(ph2_cyc)
+adcOffset = 39
+carrierFreq_MHz = 14.896010
 tx_phases = r_[0.0,90.0,180.0,270.0]
 amplitude = 1.0
 nScans = 1
```

Comment:
again, go for a nice (and labeled) grouping of things that change and things that don't